### ISSUE TITLE
Some of the benchmark test data was part of the training data of some DM models

### ISSUE THREAD

**Issue by nvanva** [Garrafao/LSCDBenchmark#92]
1) DM2 employed RuSemShift (RSS) for training and validation,  this make its comparison with other models on RuSemShift unfair. I suggest skipping DM2 results (e.g. replace with '-' or 'NA') for both RuSemShift test sets in the heatmaps.
2) Both DM1 and DM2 employ the dev set from LSCDiscovery, is it now included in our test  DWUG_es dataset? If so, both DM models have now an unfair advantage because part of the test set was used for training. The best solution would be using only test words from LSCDiscovery for testing all models. If time does not allow doing this, lets skip the results on DWUG_es for DM1 and DM2. 

Two newly added DM models (MCL, MCLen) are trained on the MCL dataset only, so their evaluation on all our test sets should be fair!

**Comment by nvanva** [comment]
One observation is very interesting: the WiC results on both RuSemShifts of DM2 are much higher that those of DM1 (because RuSemShift was part of the training data for DM2, but not DM1).

![Image](https://github.com/user-attachments/assets/d0d62ae9-faed-4213-bbfa-a78d8abe9e83)

But LSCD_Spr results are identical on the first RuSemShift and higher only on the second RuSemShift:
<img width="794" alt="Image" src="https://github.com/user-attachments/assets/b0322647-0ca9-4e2b-95d0-d2c1bf2c0bc9" />

This kind of contradicts our claim that the quality of WiC model determines the LSCD_Spr results.

**Comment by Garrafao** [comment]
> This kind of contradicts our claim that the quality of WiC model determines the LSCD_Spr results.

It's a hypothesis, not a claim. ;) Which can be refuted. Then we have something interesting to explain. We need to exclude a bug here though before proceeding.

**Comment by Garrafao** [comment]
Similar observations were reported in our paper.
https://arxiv.org/abs/2404.00176
